packages/edit-post/src/components/sidebar/post-template/form.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import { __experimentalInspectorPopoverHeader as InspectorPopoverHeader } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+import { addCard } from '@wordpress/icons';
+import { Notice, SelectControl, Button } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import PostTemplateCreateModal from './create-modal';
+
+export default function PostTemplateForm( {
+	isPostsPage,
+	selectedOption,
+	options,
+	canCreate,
+	canEdit,
+	onChange,
+	onEdit,
+	onClose,
+} ) {
+	const [ isCreateModalOpen, setIsCreateModalOpen ] = useState( false );
+
+	return (
+		<>
+			<InspectorPopoverHeader
+				title={ __( 'Template' ) }
+				help={ __(
+					'Templates define the way your content is displayed when viewing your site.'
```

Comment:
I think the first 'your' could be removed:
```suggestion
					'Templates define the way content is displayed when viewing your site.'
```